lib/bundler/source/git/git_proxy.rb

```diff
@@ -147,11 +148,14 @@ def git_retry(command)
         def git(command, check_errors = true, error_msg = nil)
           command_with_no_credentials = URICredentialsFilter.credential_filtered_string(command, uri)
           raise GitNotAllowedError.new(command_with_no_credentials) unless allow?
+          std_out, std_err, status = SharedHelpers.with_clean_git_env do
+            Open3.capture3("git #{command}")
```

Comment:
I discussed with @RochesterinNYC earlier and we didn't come up with a better idea of how to handle `stderr`